### PR TITLE
Svea step3

### PR DESCRIPTION
**Pull request by Sveouu**
- refactored premix table (in vain)
- addition of avgMnTable to host future output added via "add Mn" button (currently commented out"

### CODE REVIEW

**Comment by DanielLyleWilliamWarren** [comment]
image

Looks like you've missed a trailing conflict message!

**Comment by DanielLyleWilliamWarren** [comment]
I think the compound dropdown should also be read only! It might need some work down on step 1 to persist it first.

image

**Review by DanielLyleWilliamWarren — CHANGES_REQUESTED** [review]

File: src/app/components/dialogs/dialog.module.ts [hunk 1]
```diff
@@ -27,6 +27,7 @@ import { SectionSummaryRecurringCostReviewDialogComponent } from './sectionSumma
 import { PipesModule } from 'src/app/pipes/pipes.module';
 import { DirectivesModule } from 'src/app/directives/directives.module';
 import { ApiMetadataDialogComponent } from './apiMetadataDialog/apiMetadataDialog.component';
+
```
Comment (DanielLyleWilliamWarren) [inline src/app/components/dialogs/dialog.module.ts:30]: I know it is very minor but should avoid changes in random files...

File: src/app/components/dialogs/sectionRecurringCostReviewDialog/sectionRecurringCostReviewDialog.component.html [hunk 2]
```diff
@@ -185,7 +185,6 @@
         <tr mat-footer-row *matFooterRowDef="displayedColumns"></tr>
       </table>
     </form>
-
```
Comment (DanielLyleWilliamWarren) [inline src/app/components/dialogs/sectionRecurringCostReviewDialog/sectionRecurringCostReviewDialog.component.html:188]: I know it is very minor but should avoid changes in random files...

**Review by DanielLyleWilliamWarren — CHANGES_REQUESTED** [review]

File: src/app/pages/quickMaps/pages/costEffectiveness/interventionReview/pages/interventionMonitoringInformation/interventionMonitoringInformation.component.scss [hunk 3]
```diff
@@ -48,3 +48,6 @@ table {
 .red-icon {
   color: red;
 }
+.test {
```
[the remark this answers is not in the document]
Comment (Sveouu) [reply src/app/pages/quickMaps/pages/costEffectiveness/interventionReview/pages/interventionMonitoringInformation/interventionMonitoringInformation.component.scss:51]: no

**Review by DanielLyleWilliamWarren — APPROVED** [review]

**Pull request merged**